src/middlewares/auth.js

```diff
@@ -102,7 +105,7 @@ class Authentication {
       result.message = 'Invalid token type. Must be type Bearer';
       return result;
     }
-    const payload = await Authentication.bearer(token);
+    const payload = Authentication.bearer(token);
```

Comment:
I see you removed the async and await functionality from consumeToken and Authentication.bearer method call. Is there a reason why?

Reply:
Have you resolved this conversation?

Reply:
This is resolved